src/tests/ftest/dfuse/enospace.py

```diff
@@ -54,12 +55,14 @@ def test_enospace(self):
             file_size = 0
             while True:
                 stat_pre = os.fstat(fd.fileno())
+                if stat_pre.st_size != file_size:
+                    self.log.info('file size is %d, %s', file_size, stat_pre)
```

Comment:
FYI, there is a `self.log.warning`

Reply:
I would avoid using `self.log.warning` as it ends up marking the test status as WARN which  is a bit ambiguous.

Reply:
That's actually good news to me, the fact we're tracking warnings is a good thing.